src/components/Footer.astro

```diff
@@ -0,0 +1,77 @@
+---
+import { Image } from "astro:assets";
+import VeraIdLogo from "../assets/veraid-logo-dark.png";
+import XIcon from "../assets/icons/x.svg?raw";
+import GithubIcon from "../assets/icons/github.svg?raw";
+import RedditIcon from "../assets/icons/reddit.svg?raw";
+
+const veraidLinks = [
+  { href: "/prior-art", text: "Prior art" },
+  { href: "/ux", text: "User Experience" },
+  { href: "/architecture", text: "Architecture" },
```

Comment:
Sorry, I missed the fact that these don't follow the order of the top nav, and it's missing the link to the spec.